aliBuild

```diff
@@ -16,6 +16,10 @@ from alibuild_helpers.utilities import format, getVersion, detectArch
 from alibuild_helpers.analytics import decideAnalytics, askForAnalytics, report_screenview, report_exception, report_event
 from alibuild_helpers.analytics import enable_analytics, disable_analytics
 import traceback
+if float('%d.%d' % sys.version_info[:2]) < 2.7:
```

Comment:
How about using:

```
try:
  from collections import OrderedDict
except ImportError:
  from ordereddict import OrderedDict
```

besides that `ordereddict` is not actually a standard python module, so, how about adding it also to the dependencies in `setup.py`?